nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/DocumentSetRepositoryInit.java

```diff
@@ -28,15 +28,45 @@
  */
 public class DocumentSetRepositoryInit extends DefaultRepositoryInit {
 
-    private static final int SIZE = 10;
+    public static final String ROOT = "/default-domain/workspaces/test";
+
+    public static final int SIZE = 3;
+
+    public static int CREATED_NON_PROXY = 0;
```

Comment:
If these are variables (not constants) then they shouldn't be UPPER_CASE_WITH_UNDERSCORES.